rpc/dial_test.go

```diff
@@ -1033,7 +1033,6 @@ func TestDialFixupWebRTCOptions(t *testing.T) {
 	go func() {
 		errChan <- rpcServer.Serve(listener)
 	}()
-	test.That(t, rpcServer.Start(), test.ShouldBeNil)
```

Comment:
Start and Serve calls ans.Start(), so having both actually causes a goroutine leak